definitions/tools/add_vep_fields_to_table.cwl

```diff
@@ -4,31 +4,30 @@ cwlVersion: v1.0
 class: CommandLineTool
 label: "add VEP annotation to report"
 
-baseCommand: ["/usr/bin/python", "/usr/bin/add_annotations_to_table_helper.py"]
+baseCommand: ["vep-annotation-reporter"]
 requirements:
     - class: DockerRequirement
-      dockerPull: "mgibio/annotation_table-cwl:1.0.3"
+      dockerPull: "mgibio/vcf_annotation_tools-cwl:3.0.0"
 arguments:
-    - position: 4
-      valueFrom: $(runtime.outdir)
+    ["-o", { valueFrom: $(runtime.outdir)/variants.annotated.tsv }]
 inputs:
-    tsv:
-        type: File
-        inputBinding:
-            position: 1
     vcf:
         type: File
         inputBinding:
-            position: 2
+            position: 1
     vep_fields:
         type:
             type: array
             items: string
         inputBinding:
             prefix: ""
-            itemSeparator: ","
+            itemSeparator: " "
             separate: false
-            position: 3
+            position: 2
+    tsv:
+        type: File
+        inputBinding:
+            prefix: "-t"
```

Comment:
Is there a reason the `tsv` input moved down here? (Is it meant to now have `position: 3`?)

Reply:
tsv is no longer required in the vep-annotation-reporter, hence the removal of the `position` inputBinding. The order reflects the one in the tool, with required arguments on top.

Reply:
If it's optional, should the type be `type: File?` instead?  It could still have a `position` specified for those times when the file is supplied.

Reply:
Yes, good catch. Why would we want the `position` inputBinding though, when the argument of the tool is not positional?

Reply:
It doesn't necessarily matter if the tool doesn't care. If it's required to be before the positional arguments I might want to make that explicit.  (There's a default position of `0` so it'll happen to be first in this circumstance.)

Reply:
I'm pretty sure the tool doesn't care.